modules/apps/asset/asset-list-web/test/js/components/VariationsNav.test.js

```diff
@@ -225,7 +223,7 @@ describe('VariationsNav With segments', () => {
 
 describe('VariationsNav', () => {
 	beforeEach(() => {
-		global.Liferay.Util.openToast = jest.fn();
+		global.Liferay.component = jest.fn();
```

Comment:
I saw you mentioned that this fixed a test that was failing, but any idea why we would need this in the first place? What is calling Liferay.component now that wasn't before?

Reply:
I believe this was caused by the switch to using imported `openToast`. Shouldn't be relevant anymore with the new mock.